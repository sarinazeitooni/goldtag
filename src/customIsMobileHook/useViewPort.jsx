import React from "react";
const useViewPort = () => {
    const [width, setWidth] = React.useState(typeof window!=="undefined" && window.innerWidth);

    React.useEffect(() => {
        const handleWindowResize = () => setWidth(window.innerWidth);
        window.addEventListener("resize", handleWindowResize);
        return () => window.removeEventListener("resize", handleWindowResize);
    }, []);

    return width<700;
}
export default useViewPort;