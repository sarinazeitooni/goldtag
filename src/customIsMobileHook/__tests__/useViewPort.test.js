import React from 'react';
import { shallow } from 'enzyme';
import useViewPort from "../useViewPort";
describe("useViewPort hook" , ()=>{
    it("it should render",()=>{
        const container = shallow(<useViewPort/>)
        expect(container).toMatchSnapshot();
    });
});