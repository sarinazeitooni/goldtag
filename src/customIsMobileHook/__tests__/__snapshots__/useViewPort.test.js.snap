// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`useViewPort hook it should render 1`] = `ShallowWrapper {}`;
